pkg/clusteragent/admission/mutate/auto_instrumentation_test.go

```diff
@@ -685,19 +714,70 @@ func TestInjectAll(t *testing.T) {
 	}
 }
 
+func injectAllEnvs() []corev1.EnvVar {
+	return []corev1.EnvVar{
+		{
+			Name:  "PYTHONPATH",
+			Value: "/datadog-lib/",
+		},
+		{
+			Name:  "RUBYOPT",
+			Value: " -r/datadog-lib/auto_inject",
+		},
+		{
+			Name:  "NODE_OPTIONS",
+			Value: " --require=/datadog-lib/node_modules/dd-trace/init",
+		},
+		{
+			Name:  "JAVA_TOOL_OPTIONS",
+			Value: " -javaagent:/datadog-lib/dd-java-agent.jar",
```

Comment:
Should JAVA_TOOL_OPTIONS be the same as in https://github.com/DataDog/datadog-agent/pull/17320/files#diff-09392d73a78b8062441e51d5a721f58eaf5e8c47a20ffe5243f7752b82a06901R37?

Reply:
yep, it should be